src/cirrus/build.py

```diff
@@ -12,16 +12,55 @@
 import os
 import sys
 from argparse import ArgumentParser
+import pluggage.registry
 
 from cirrus.documentation_utils import build_docs
-from cirrus.environment import cirrus_bin, is_anaconda, repo_directory
-from cirrus.configuration import load_configuration, get_pypi_auth
-from cirrus.pypirc import PypircFile
+from cirrus.environment import is_anaconda
```

Comment:
This is actually a legit method, it falls back to using it to determine if you are using conda or not if not plugin is provided